### ISSUE TITLE
Certifications in Progress shows false certification opportunity

### ISSUE THREAD

**Issue by YCSU** [udacity/reviewer-dashboard-feedback#31]
There are six false certification opportunities appears on the certifications page and to do
![2018-02-09-201948_2560x1440_scrot](https://user-images.githubusercontent.com/7635036/36027753-7781fee8-0dd7-11e8-8631-05456e783614.png)

**Comment by YCSU** [comment]
[Uploading 2018-02-09-201948_2560x1440_scrot.png...]

**Comment by kartikarora** [comment]
#7 already done

**Comment by YCSU** [comment]
These false items still appear on my dashboard

**Comment by romcabrera** [comment]
Same here. I'm seeing "To Do" Onboarding opportunities, when i click on them they do not appear. This, after #7 was marked as "Done".

**Comment by romcabrera** [comment]
Update: this is fixed now.
